src/components/views/settings/shared/SettingsSubsectionHeading.tsx

```diff
@@ -25,7 +25,7 @@ export interface SettingsSubsectionHeadingProps extends HTMLAttributes<HTMLDivEl
 
 export const SettingsSubsectionHeading: React.FC<SettingsSubsectionHeadingProps> = ({ heading, children, ...rest }) => (
     <div {...rest} className="mx_SettingsSubsectionHeading">
-        <Heading className="mx_SettingsSubsectionHeading_heading" size="h3">
+        <Heading className="mx_SettingsSubsectionHeading_heading" size="h4">
```

Comment:
Does this not violate https://dequeuniversity.com/rules/axe/4.4/heading-order

Reply:
Good shout! I've added a parameter to be able to independently change the appearance from the heading level